### DISCUSSION TITLE
Consider renaming manifest key `pypi-dependencies`

### DISCUSSION THREAD

**Discussion by Hofer-Julian** [prefix-dev/pixi#1178] [Ideas]
### Problem description

I feel like the name `pypi-dependencies` is pretty confusing: https://pixi.sh/dev/reference/configuration/#pypi-dependencies-beta-feature

Naming it after PyPI implies to me that the packages come from https://pypi.org/, but that isn't necessarily the case.
I liked `pip-dependencies` but @tdejager told me that isn't exactly an improvement.
What is happening is that you either install a binary or source distribution. Maybe `whl-sdist-dependencies` would work?

**Comment by ruben-arts** [comment]
The though was: 
`python-dependencies` is unfair as conda packages can contain python packages.
`py-dependencies` insists previous option.

We looked into what a convention would be but nothing specifies a different word for a `python` package. 

`pip-dependencies` is equally weird as it makes the user think it is installed by `pip`. 

There are a few server options that take over the `pypi` name to so we decided to make everything in the whole code-base `pypi`.

I don't see a good reason to do a change at the moment. Feel free to share some cases where users are confused or think they have a better option. 

I'm moving this to a discussion as I understand we should have an open discussion but it doesn't fit in the issues.
